core/server/worker/src/test/java/alluxio/worker/block/MonoBlockStoreCommitBlockTest.java

```diff
@@ -0,0 +1,159 @@
+            // LocalCacheManager line 107 🤔
+            Thread.sleep(1000);
+            writer.append(buf);
+        } catch (Exception e) {
+            System.out.println(e);
+        }
+        mMonoBlockStore.registerBlockStoreEventListener(listener);
+        mMonoBlockStore.commitBlock(sessoinId, blockId, false);
+
+        verify(listener).onCommitBlockToLocal(anyLong(), any(BlockStoreLocation.class));
+        verify(listener).onCommitBlockToMaster(anyLong(), any(BlockStoreLocation.class));
+    }
+
+    @Test
+    public void CommitLocalSuccessandCommitMasterFail() {
+        try {
+            doAnswer((i) -> {
+                throw new AlluxioStatusException(Status.UNAVAILABLE);
+            }).when(mockedBlockMasterClient).commitBlock(anyLong(), anyLong(), anyString(), anyString(), anyLong(), anyLong());
+        } catch (Exception e) {
+            System.out.println(e);
+        }
+        mTieredBlockStore = new TieredBlockStore(mBlockMetadataManager, mBlockLockManager);
+
+        mMonoBlockStore = new MonoBlockStore(mTieredBlockStore, mockedBlockMasterClientPool, mock(UfsManager.class), new AtomicReference<>(1L));
+
+        mMonoBlockStore.createBlock(sessoinId, blockId, FIRST_TIER, new CreateBlockOptions(null, null, 64));
+        byte[] data = new byte[64];
+        Arrays.fill(data, (byte) 1);
+        ByteBuffer buf = ByteBuffer.wrap(data);
+        try (BlockWriter writer = mMonoBlockStore.createBlockWriter(1L, blockId)) {
+            // LocalCacheManager line 107 🤔
+            Thread.sleep(1000);
+            writer.append(buf);
+        } catch (Exception e) {
+            System.out.println(e);
+        }
+        mMonoBlockStore.registerBlockStoreEventListener(listener);
+        assertThrows(RuntimeException.class, () -> {
+            mMonoBlockStore.commitBlock(sessoinId, blockId, false);
+        });
+
+        verify(listener).onCommitBlockToLocal(anyLong(), any(BlockStoreLocation.class));
+        verify(listener, never()).onCommitBlockToMaster(anyLong(), any(BlockStoreLocation.class));
+    }
+
+
+    @Test
+    public void CommitLocalFailandCommitMasterSuccess() {
+        mTieredBlockStore = spy(new TieredBlockStore(mBlockMetadataManager, mBlockLockManager));
+        doAnswer((i) -> {
+            throw new RuntimeException();
+        }).when(mTieredBlockStore).commitBlockInternal(anyLong(), anyLong(), anyBoolean());
+
+        mMonoBlockStore = new MonoBlockStore(mTieredBlockStore, mockedBlockMasterClientPool, mock(UfsManager.class), new AtomicReference<>(1L));
+
+        mMonoBlockStore.createBlock(sessoinId, blockId, FIRST_TIER, new CreateBlockOptions(null, null, 64));
+        byte[] data = new byte[64];
+        Arrays.fill(data, (byte) 1);
+        ByteBuffer buf = ByteBuffer.wrap(data);
+        try (BlockWriter writer = mMonoBlockStore.createBlockWriter(1L, blockId)) {
+            // LocalCacheManager line 107 🤔
+            Thread.sleep(1000);
```

Comment:
why sleep here?

Reply:
PageTest will fail without this sleep, and MonoTest just copied some code from PageTest..